common/models/access-token.js

```diff
@@ -7,6 +7,9 @@
  * Module Dependencies.
  */
 
+var SG = require('strong-globalize');
+var g = SG();
```

Comment:
I think it's worth extracting these two lines to a shared file in lib folder. That way we can easily add additional initialization in one place.

```js
/* here in this file */
var g = require('../../lib/globalize');

/* lib/globalize */
var SG = require('strong-globalize');
var g = SG();
// later we can add any shared initialization here
module.exports = g;
```

Thoughts?